oeps/processes/oep-0001.rst

```diff
@@ -489,32 +489,51 @@ OEPs may include auxiliary files such as diagrams. Such files must be added to
 an oep-XXXX/ directory, where "XXXX" is the OEP number. Include original diagrams
 alongside image files, to make it easy for others to update the OEP in the future.
 
+Change History
+--------------
+
+For every change (including the initial document creation), include an entry in
+a "Change History" section modeled off the one below. A Change History entry
+should include three parts: the date of the change, a very brief summary of
+changes made, and a link to the pull request where the discussion and approval
+took place.
```

Comment:
```suggestion
took place.  The changes should be ordered such that the most recent change is at the top of the list.
```
Happy for it to go the other direction as well but right now, we don't prescribe this and it has resulted in some confusion.  And some change histories that have been partially out of order as some people add changes to the top while others add them to the bottom.

Reply:
Definitely let's pick a direction! :)   Most recent at the top is good.